spec/models/index_section_spec.rb

```diff
@@ -1,60 +1,35 @@
 RSpec.describe IndexSection do
+  let(:user) { create(:user) }
+  let(:index) { create(:index, user: user) }
+
   it "should autofill post section order when not specified" do
-    index = create(:index)
     section = create(:index_section, index: index)
-    post2 = create(:post, user: index.user)
-    post0 = create(:post, user: index.user)
-    post1 = create(:post, user: index.user)
-    section.posts << post0
-    section.posts << post1
-    section.posts << post2
+    post2 = create(:post, user: user)
+    post0 = create(:post, user: user)
+    post1 = create(:post, user: user)
+    posts = [post0, post1, post2]
```

Comment:
```suggestion
    posts = create_list(:post, 3, user: user)
```
? (I'll stop posting this suggestion, but there might be other locations with it)